db/migrate/20151227195814_devise_create_users.rb

```diff
@@ -0,0 +1,44 @@
+class DeviseCreateUsers < ActiveRecord::Migration
+  def self.up
+    create_table(:users) do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.